File: .net 6.0/ListenAI.LLM/Models/Response/CompletionChoice.cs
using Newtonsoft.Json;

namespace ListenAI.LLM.Models.Response; 

public class CompletionChoice {
    /// <summary>
    /// 会话结束原因
    /// </summary>
    [JsonProperty("finish_reason")]
    public string FinishReason { get; set; }
    
    /// <summary>
    /// 消息排序
    /// </summary>
    [JsonProperty("index")]
    public int Index { get; set; }
    
    /// <summary>
    /// 消息文本内容
    /// </summary>
    [JsonProperty("message")]
    public SparkMessage Message { get; set; }
}

File: java 17/llm/src/main/java/com/listenai/llm/models/response/CompletionChoice.java
package com.listenai.llm.models.response;

import com.alibaba.fastjson2.annotation.JSONField;
import com.listenai.llm.models.request.SparkMessage;

public class CompletionChoice {
    @JSONField(name = "finish_reason")
    private String finishReason;

    @JSONField(name = "index")
    private int index;

    @JSONField(name = "message")
    private SparkMessage message;

    public String getFinishReason() {
        return finishReason;
    }

    public void setFinishReason(String finishReason) {
        this.finishReason = finishReason;
    }

    public int getIndex() {
        return index;
    }

    public void setIndex(int index) {
        this.index = index;
    }

    public SparkMessage getMessage() {
        return message;
    }

    public void setMessage(SparkMessage message) {
        this.message = message;
    }
}
